web/src/routes/PoisPage.tsx

```diff
@@ -81,14 +83,20 @@ const PoisPage = ({ cityCode, languageCode, cityModel, pathname, languages }: Ci
     }
   })
 
+  const selectFeature = (feature: PoiFeature | null, snapPoint?: number) => {
+    setCurrentFeature(feature)
+    if (snapPoint !== undefined) {
+      sheetRef.current?.snapTo(({ maxHeight }) => getSnapPoints(maxHeight)[snapPoint]!)
+    }
+  }
```

Comment:
I find the snap point logic kind of hard to understand. When do you pass a snap point index? And when don't you? Sometimes you pass `0`, sometimes `1` and sometimes nothing.

Reply:
you only pass a snappoint if you want to change the current snappoint of the bottom action sheet